x/pocketcore/keeper/proof.go

```diff
@@ -73,8 +76,24 @@ func (k Keeper) SendProofTx(ctx sdk.Ctx, n client.Client, node *pc.PocketNode, p
 		if !found {
 			ctx.Logger().Error(fmt.Sprintf("an error occurred creating the proof transaction with app %s not found with evidence %v", evidence.ApplicationPubKey, evidence))
 		}
+
+		// There is a potential race condition where the evidence is not sealed while submitting a claim
```

Comment:
Do you have any idea why this wasn't a major concern in the past?

Reply:
Generally whenever the claim is sealed, relays stop coming in and so there is some non deterministic timing in which the race condition could happen. If relays keep coming in (i.e session rollover), the problem becomes more prominent.

We did find some on chain tx's to show that this did periodically happen even without session rollover though.

Reply:
![image](https://github.com/pokt-network/pocket-core/assets/99137075/bed47b29-7fa5-4e8b-bb3a-d1a010204ae7)
You can go [here ](https://www.poktscan.com/node_runner?domain=poktscan.cloud&tab=transfers)and filter the transactions by "Proof" and "Failed", and put the start date before ~ the 10th and it will show some of the failed submissions even with session rollover disabled.

Reply:
Was running into some issues but reached out to poktscan here: https://discord.com/channels/854406364931686400/950799595171086377/1131367539667107872